src/server/api/metadata.js

```diff
@@ -119,5 +119,15 @@ export default function metadataRouter() {
         })
     );
 
+    logger.info("adding the GET /api/unified-astronomy-thesaurus handler");
```

Comment:
This log message isn't terribly important but isn't accurate and should probably get updated